src/spec-node/singleContainer.ts

```diff
@@ -193,6 +193,9 @@ async function buildAndExtendImage(buildParams: DockerResolverParameters, config
 				args.push('--load'); // (short for --output=docker, i.e. load into normal 'docker images' collection)
 			}
 		}
+		if (buildParams.buildxCacheTo) {
+			args.push('--cache-to', buildParams.buildxCacheTo);
```

Comment:
This also works without BuildKit, doesn't it? If so, you can move it out of the if/else.

Reply:
`--cache-to` is an option for `buildx build` command. I think it does not work without BuildKit.

e.g.

```.sh
❯ docker build --cache-to some-cache .
unknown flag: --cache-to
See 'docker build --help'.
```